mods/xpanes/init.lua

```diff
@@ -160,6 +159,19 @@ xpanes.register_pane("pane", {
 	}
 })
 
+xpanes.register_pane("obsidian_pane", {
+	description = "Obsidian Glass Pane",
+	textures = {"default_obsidian_glass.png","xpanes_pane_half.png","xpanes_obsidian.png"},
+	inventory_image = "default_obsidian_glass.png",
+	wield_image = "default_obsidian_glass.png",
+	sounds = default.node_sound_glass_defaults(),
+	groups = {snappy=2, cracky=3, oddly_breakable_by_hand=3},
```

Comment:
'oddly_breakable_by_hand' needs removing.

Reply:
this is weird too, I had already removed that in a prior commit.

Reply:
You added it again in a later one.